### DISCUSSION TITLE
Batch Tagging

### DISCUSSION THREAD

**Discussion by coreyward** [sanity-io/sanity-plugin-media#25] [Ideas]
The Media plugin currently has the UI for batch selections, but the only available actions on a batch selection are “Deselect” and “Delete”. It would be exceedingly useful to get tagging support added to these batch selections, though, especially when adding a slew of images that match some specific type (e.g. logos, blog author headshots, or icons).

**Comment by robinpyon** [comment]
Hey @coreyward – this is something that's been cooking for some time and is pretty close to release! You can actually check it out on the tagged `beta` branch (currently `1.1.0-beta.12`), and it's been heavily inspired by how batch tagging works in Adobe Lightroom. Give it a go – would appreciate any feedback you have here.

**Reply by coreyward** [reply]
I had a chance to play with this finally and it's not super evident that it is available, but it works pretty well once you know about it. Thanks!
